WordPress-Docs/ruleset.xml

```diff
@@ -116,4 +116,6 @@
 		<!-- WP allows @todo's in comments -->
 		<exclude name="Generic.Commenting.Todo.TaskFound"/>
 	</rule>
+
+	<rule ref="WordPress.Commenting.NoReturnVoid"/>
```

Comment:
A comment could be added here referencing the handbook rule that this sniff relates to.